staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/matching/matching_test.go

```diff
@@ -424,11 +426,38 @@ func TestMatcher(t *testing.T) {
 			attrs:         admission.NewAttributesRecord(nil, nil, schema.GroupVersionKind{"autoscaling", "v1", "Scale"}, "ns", "name", schema.GroupVersionResource{"extensions", "v1beta1", "deployments"}, "", admission.Create, &metav1.CreateOptions{}, false, nil),
 			expectMatches: false,
 		},
+		{
+			name: "bindingMatch treat empty ResourceRules as match",
+			criteria: &v1alpha1.MatchResources{
+				NamespaceSelector: &metav1.LabelSelector{},
+				ObjectSelector:    &metav1.LabelSelector{},
```

Comment:
Why the empty selectors?

Reply:
That is the default if those fields are unset